views/conditions/fungal-nail-infection.html

```diff
@@ -6,6 +6,10 @@
   <div id="local-header">
     <div class="header-local-container">
       <h1>{{ pagetitle }}</h1>
+      <div class="measure">
```

Comment:
I reckon this 'measure' class is redundant, as the grid itself should determine line length, not a 'magic number'